canonical-web-and-design/ubuntu.com
# WIP - Build landscape form

## Done

- Built as per [copy doc](https://docs.google.com/document/d/1ZgovfWgMFPz2mnSoprhrdYuCiedai-LKIfR8FHKSr3w/edit)
- **NOTE: Waiting on external input for QA**

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/landscape
    - Be sure to test on mobile, tablet and desktop screen sizes
- Click on the 'contact us' button and engage with the form
- Make sure that required input (everything except the last checkbox) behaves accordingly 

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5148
